web/tests/topic_list.test.cjs

```diff
@@ -0,0 +1,72 @@
+"use strict";
+
+const assert = require("node:assert/strict");
+
+const {mock_esm, zrequire} = require("./lib/namespace.cjs");
+const {run_test, noop} = require("./lib/test.cjs");
+
+mock_esm("../src/people.ts", {
+    maybe_get_user_by_id: noop,
+});
+
+const all_messages_data = zrequire("all_messages_data");
+const {set_realm} = zrequire("state_data");
+const stream_data = zrequire("stream_data");
+const stream_topic_history = zrequire("stream_topic_history");
+const topic_list = zrequire("topic_list");
+
+set_realm({});
+
+function test(label, f) {
+    run_test(label, (helpers) => {
+        f(helpers);
+    });
+}
+
+test("is_full_topic_history_available", ({override}) => {
```

Comment:
Note: The things `"is_complete_for_stream_id"` was verifying in `stream_topic_history.test.cjs` was already getting verified in `all_topics_in_cache`, so merged them into one here. And renamed to `is_full_topic_history_available`.